internal/controllers/common_controller.go

```diff
@@ -110,9 +110,7 @@ func setDeprecationStatusesUnknown(conditions *[]metav1.Condition, message strin
 // setProgressingStatusConditionSuccess sets the progressing status condition to false for a successful install or upgrade.
 func setProgressingStatusConditionSuccess(conditions *[]metav1.Condition, message string, generation int64) {
 	apimeta.SetStatusCondition(conditions, metav1.Condition{
-		Type:               ocv1alpha1.TypeProgressing,
```

Comment:
If ClusterExtension controller is calling these functions, let's make sure they remain unchanged. That again might mean moving types from `extension_types.go` into `clusterextension_types.go`.

Reply:
I saw. I needed to move some of those over the other. I'm getting some test errors. Now I need to work through them. Thanks for your assistance @joelanford ! I appreciate it.

Reply:
I totally misinterpreted this interaction. We can remove all these functions that have to do with setting the `Progressing` condition and _also_ remove the condition type. 

The condition type and functionality can be added back as part of #842